### PR TITLE
Fix preview screen opening twice when no filament is loaded

### PR DESCRIPTION
**Pull request by Chleba**
BFW-1293

### CODE REVIEW

File: src/gui/screen_print_preview.cpp [hunk 1]
```diff
@@ -204,7 +208,7 @@ bool screen_print_preview_data_t::gcode_file_exists() {
 
 //FIXME simple solution not to brake functionality before release
 //rewrite later
-static bool suppress_draw = false;
+// static bool suppress_draw = false;
```
Comment (DRracer) [inline src/gui/screen_print_preview.cpp:211]: please remove unused code

File: src/gui/screen_print_preview.cpp [hunk 2]
```diff
@@ -227,7 +231,7 @@ void screen_print_preview_data_t::windowEvent(EventLock /*has private ctor*/, wi
             break;
         case Response::No: //NO - cancel
             Screens::Access()->Close();
-            suppress_draw = false;
+            // suppress_draw = false;
```
Comment (DRracer) [inline src/gui/screen_print_preview.cpp:234]: please remove unused code

File: src/gui/screen_print_preview.hpp [hunk 3]
```diff
@@ -55,6 +55,9 @@ struct screen_print_preview_data_t : public AddSuperWindow<window_frame_t> {
     window_text_t print_label;
     window_icon_button_t back_button;
     window_text_t back_label;
+    // static bool suppress_draw = false;
```
Comment (DRracer) [inline src/gui/screen_print_preview.hpp:58]: please remove unused code

File: src/gui/screen_print_preview.cpp [hunk 4]
```diff
@@ -163,6 +163,8 @@ static void print_button_press() {
     }
 }
 
+// screen_print_preview_data_t::suppress_draw = false;
```
Comment (DRracer) [inline src/gui/screen_print_preview.cpp:166]: please remove unused code

**Review by DRracer — CHANGES_REQUESTED** [review]
After removal of unused code (which has been superseded by the new functionality) this PR may be merged

**Review by RadekVana — APPROVED** [review]

**Comment by Chleba** [comment]
All unused code was removed - please approve @DRracer

**Review by DRracer — APPROVED** [review]

**Pull request merged**